### PR TITLE
Fixes `expecting an AddressLocation` bug

### PR DESCRIPTION
**Pull request by ph0ph0**
Addresses: https://github.com/onflow/flow-js-testing/issues/191

## Description

---

For contributor use:

- [ ] Targeted PR against `master` branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work
- [ ] Code follows the [standards mentioned here](https://github.com/onflow/flow-js-testing/blob/master/CONTRIBUTING.md#styleguides)
- [ ] Updated relevant documentation
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

File: src/interaction.js [hunk 1]
```diff
@@ -178,7 +178,7 @@ export const executeScript = async (...props) => {
     try {
       const extractor = extractParameters("script")
       const {code, args, limit} = await extractor(props)
-
+      console.log(`About to run extractor with props: ${JSON.stringify(props)}`)
```
Comment (ph0ph0) [inline src/interaction.js:181]: Adding this log statement here fixes the issue. Not sure why this is, possibly something to with bundling/minified version??

**Comment by adbario** [comment]
@ph0ph0 Here's already a fix for this, just waiting for it to get approved 👍

**Comment by gregsantos** [comment]
Think we can close this as https://github.com/onflow/flow-js-testing/pull/190 has been merged

**Comment by gregsantos** [comment]
Thanks for your contribution @ph0ph0 !
Closing this as completed via https://github.com/onflow/flow-js-testing/pull/190 by @adbario! 🚀

**Pull request closed without merging**